foundations/intro-to-css/02-class-id-selectors/style.css

```diff
@@ -0,0 +1,19 @@
+.odd {
+    background-color: rgb(255, 192, 203);
+    font-family: Verdana, "DejaVu Sans", sans-serif;
+}
+
+#second {
+    font-size: 36px;
+    color: #0000ff;
+}
+
+#third {
+    font-size: 24px;
```

Comment:
Just a comment for the future: if `#third` and `#fourth` have the same font size by coincidence, then your solution is the best way to do it. But if they are actually connected, and you'd like to keep them at the same font size in future changes, you would want to extract out this logic:

```
#third,
#fourth {
    font-size: 24px;
}
```